src/components/Identicon.js

```diff
@@ -1,27 +1,21 @@
 class Identicon { // eslint-disable-line no-unused-vars
     /**
      * @param {string} [address]
-     * @param {HTMLImageElement} [el]
+     * @param {HTMLDivElement} [$el]
      */
-    constructor(address, el) {
+    constructor(address, $el) {
         this._address = address;
 
-        /** @type {HTMLImageElement} */
-        this.$el = el || this.getElement();
+        this.$el = this._createElement($el);
```

Comment:
You made the `_createElement` method static below in `PassphraseInput`, but not here. We should have a standard.